administrator/language/en-GB/plg_system_languagefilter.ini

```
@@ -5,6 +5,7 @@
 
 PLG_SYSTEM_LANGUAGEFILTER="System - Language Filter"
 PLG_SYSTEM_LANGUAGEFILTER_BROWSER_SETTINGS="Browser Settings"
+PLG_SYSTEM_LANGUAGEFILTER_ERROR_NO_CONTENT_LANGUAGE="You have no content language enabled, but have enabled the Language Filter plugin. This will result in numerous errors!"
```

```suggestion
PLG_SYSTEM_LANGUAGEFILTER_ERROR_NO_CONTENT_LANGUAGE="You have no content language published, but have enabled the Language Filter plugin. This will result in numerous errors!"
```
I agree with Brian that the text should be changed. @Hackwar Please check.